lib/Dialect/TritonGPU/IR/Types.cpp

```diff
@@ -88,6 +88,28 @@ LogicalResult MemDescType::verify(function_ref<InFlightDiagnostic()> emitError,
   return success();
 }
 
+llvm::TypeSize
+MemDescType::getTypeSizeInBits(const DataLayout &datalayout,
```

Comment:
using llvm data layout seems a bit of an overkill, could we just skip it like we do for other allocs?

Reply:
Yeah I'm not a huge fan of it. Let me just strip this all out.